src/core/Akka.Cluster/ClusterSettings.cs

```diff
@@ -57,8 +57,7 @@ public ClusterSettings(Config config, string systemName)
 
             Roles = cc.GetStringList("roles").ToImmutableHashSet();
             MinNrOfMembers = cc.GetInt("min-nr-of-members");
-            //TODO:
-            //_minNrOfMembersOfRole = cc.GetConfig("role").Root.GetArray().ToImmutableDictionary(o => o. )
```

Comment:
Already implemented this feature a long time ago, so the TODO was no longer needed.